### PR TITLE
Make server feature serve API usage analytics.

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @rmmh

### CODE REVIEW

File: mungegithub/features/server.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ func (s *ServerFeature) Name() string {
 
 // Initialize will initialize the feature.
 func (s *ServerFeature) Initialize(config *github.Config) error {
+	config.ServeDebugStats("/stats")
```
Comment (rmmh) [inline mungegithub/features/server.go]: Does this work? config.ServeDebugStats uses http.HandleFunc, but I think s.ConcurrentMux uses a non-singleton http handler?

Comment (cjwagner) [reply mungegithub/features/server.go]: See the next line. `s.ConcurrentMux` wraps the `http.DefaultServeMux` which is the mux that `http.HandleFunc` will register to.

**Pull request merged**